Libraries/oneMKL/random_sampling_without_replacement/lottery.cpp

```diff
@@ -13,6 +13,7 @@
 *******************************************************************************/
 
 #include <iostream>
+#include <unordered_set>
```

Comment:
I assume we need to include <algorithm> instead of unordered_set for  adjacent_find

Reply:
however the logic with unordered_set seems a bit easier for me (not insisting on the change), just for an example:
```
for(size_t i = 0; i < num_exp; ++i) {
    unordered_set<size_t> nums;
    for(size_t j = 0; j < m; j++) {
        size_t curr =  result_ptr[m * i + j];
        if(nums.find(curr) != nums.end()) {
            return 1;
        }
        nums.insert(curr);
    }
```